Modules/rat_command.py

```diff
@@ -52,86 +51,46 @@ class NameCollisionException(CommandException):
 # character/s that must prefix a message for it to be parsed as a command.
 prefix = config['commands']['prefix']
 
-# Pydle bot instance.
-bot: BasicClient = None
 
-
-async def trigger(message: str, sender: str, channel: str):
+async def trigger(ctx):
     """
-    Invoke a command, passing args and kwargs to the called function
-    :param message: triggers message to invoke
-    :param sender: author of triggering message
-    :param channel: channel of triggering message
-    :return: bool command
+
+    Args:
+        ctx (Context):
+
+    Returns:
+
     """
-    if bot is None:
-        # someone didn't set me.
-        raise CommandException(f"Bot client has not been created"
-                               f" or not handed to Commands.")
-    if message.strip() == "":
-        return
-
-    words, words_eol = _split_message(message.lstrip(prefix))
-    if message.startswith(prefix):
-        if words[0].casefold() in _registered_commands.keys():
+
+    if ctx.words_eol[0] == "":
+        return  # empty message, bail out
+
+    if ctx.prefixed:
```

Comment:
I know it doesn't change anything, but just for my peace of mind, could you change this to `elif`?